subnet/kube/kube.go

```diff
@@ -457,9 +449,14 @@ func (ksm *kubeSubnetManager) Name() string {
 	return fmt.Sprintf("Kubernetes Subnet Manager - %s", ksm.nodeName)
 }
 
-// Set Kubernetes NodeNetworkUnavailable to false when starting
+// CompleteLease Set Kubernetes NodeNetworkUnavailable to false when starting
 // https://kubernetes.io/docs/concepts/architecture/nodes/#condition
-func (ksm *kubeSubnetManager) setNodeNetworkUnavailableFalse(ctx context.Context) error {
+func (ksm *kubeSubnetManager) CompleteLease(ctx context.Context, lease *subnet.Lease, wg *sync.WaitGroup) error {
+	if !ksm.setNodeNetworkUnavailable {
```

Comment:
You forgot to call `err = ksm.setNodeNetworkUnavailableFalse(ctx)` when ksm.setNodeNetworkUnavailable = true

Reply:
You are right, I'm stupid